frontend/public/components/utils/volume-icon.jsx

```diff
@@ -1,17 +1,22 @@
 import * as _ from 'lodash-es';
 import * as React from 'react';
 import {VolumeSource} from '../../module/k8s/pods';
+import { ResourceLink } from './resource-link';
 
-export const VolumeIcon = ({kind}) => {
+export const VolumeIcon = ({kind, name}) => {
```

Comment:
We should rename this. This is no longer just an icon. It should be a component that renders the entire value, not just the icon now (since we have to render links in some cases).

Reply:
Yes, I agree. I didn't want to change the name without asking you first. What do you suggest we should rename it to?